### PR TITLE
Prettier walls

### LINKED ISSUE #138: Add glow to the walls
I wanted to make this.

### PR DESCRIPTION
**Pull request by rexim**
Close #138

### CODE REVIEW

**Comment by rexim** [comment]
Wow! The coverage even increased! Well, good for me. ;)

File: pacmacs-image.el [hunk 1]
```diff
@@ -65,20 +75,39 @@
                            (if bit 1 0))))
     result))
 
-(defun pacmacs--bits-to-xpm (bits width height)
+(defun pacmacs--generate-xpm-palette (palette)
+  (let* ((n (length palette))
+         (palette-indices (number-sequence 0 (1- n))))
+    (->> (-map #'identity palette)
```
Comment (rexim) [inline pacmacs-image.el]: Hm... palette is a list everywhere now. I believe I can get rid of that conversion.

Comment (rexim) [inline pacmacs-image.el]: It was needed to convert vector to list. :)

File: pacmacs-image.el [hunk 2]
```diff
@@ -91,53 +120,117 @@
...
+
 (defun pacmacs--create-wall-tile (width height
-                                  bottom right
-                                  top left
-                                  left-upper right-upper
-                                  left-bottom right-bottom)
+                                        bottom right
+                                        top left
+                                        left-upper right-upper
+                                        left-bottom right-bottom)
```
Comment (ForNeVeR) [inline pacmacs-image.el]: Why did you indented that further?

Comment (rexim) [inline pacmacs-image.el]: It's not me! It's Emacs! Its identation thinks that the first element of the list is a function name and everything else is aligned as the function's arguments.

It's not a big deal, actually, but I'll fix it anyway. Thanks!

**Comment by ForNeVeR** [comment]
Aside from the issues already mentioned, this code looks good to me.

**Comment by rexim** [comment]
ATT on Linux: `~0.008s`

**Comment by rexim** [comment]
ATT on Windows: `~0.020s`

**Comment by rexim** [comment]
Nothing unusual. Everything seems to be fine. Mergin the request.

**Pull request merged**